flow360/examples/base_test_case.py

```diff
@@ -62,6 +62,11 @@ def surface_json(self):
     def surface_yaml(self):
         """surface_yaml"""
 
+    @property
+    @abstractmethod
+    def extra(self):
+        """extra"""
+
```

Comment:
If we agree to call the conversion function to generate the betdisk related json files (related comment: https://github.com/flexcompute/Flow360/pull/553#discussion_r1856986326), the modification would be unnecessary.

Reply:
I think we shouldn't call the conversion function in this tutorial